client/src/App.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
+import Login from './pages/login';
+// import Flight from './pages/flights';
+import AddFlight from './pages/addFlight';
+import UpdateFlight from './pages/updateFlight';
+import './App.css';
 
 class App extends Component {
   render() {
     return (
```

Comment:
This return should produce an error.

Reply:
Sorry it's not this line, this should be fine. It's another line. I'll tag you in a second.

Reply:
@Balsam-Faysal Sorry, my fault.